docs/ABOUT.md

```diff
@@ -0,0 +1,46 @@
+[Dart](https://en.wikipedia.org/wiki/Dart_(programming_language)) is a general-purpose computer programming language that is:
+ 1. [Concurrent](https://en.wikipedia.org/wiki/Concurrent_computing)
+ 2. [Class-Based](https://en.wikipedia.org/wiki/Class-based_programming)
+ 3. [Object-Oriented](https://en.wikipedia.org/wiki/Object-oriented_programming)
+
+Dart applications are typically compiled to [bytecode](https://en.wikipedia.org/wiki/Bytecode) that can run on any [Dart virtual machine](https://en.wikipedia.org/wiki/Dart_virtual_machine) (JVM) regardless of computer architecture.
+
+The primary concepts baked into the making of the language were:
+
+ - It is simple, object-oriented, and familiar.
+ - It provides tools out of the box to help development efforts.
+ - It is architecture-neutral and portable.
+ - It executes with high performance for practical applications and not just benchmarks.
+
+The language derives its syntax from Java and JavaScript; as well as Smalltalk.
+
+## Why Learn Dart?
+
+- DART IS FROM GOOGLE.
+It is the programming language between many of their produces like AdSense.
+It has grown to provide solutions for web-based content, and mobile apps
+
+- [Write once, run anywhere](https://en.wikipedia.org/wiki/Write_once,_run_anywhere) policy: Programs written in Dart are compiled into machine-independent bytecodes (or JavaScript for web apps), so it can run on any Dart platform.
+Basically, Dart is [Platform Independent](https://en.wikipedia.org/wiki/Cross-platform#Platform-independent_software)
+
+- Dart has a solid API:
+Dart has built-in [application programming interfaces](https://en.wikipedia.org/wiki/Application_programming_interface) (APIs) for I/O, networking, data structures, concurrency, and many other functions.
+Standard libraries can be supplemented by open source libraries found on [Pub](https://pub.dartlang.org).
+
+- Standardization:
+If you need to write a web/enterprise app, or you need data persistence or you need to do cryptography, no problem!
+Dart is all you require.
+There are many standard tools/practices that extend the [Dart specification](https://www.dartlang.org/guides/language/spec) , thanks to the [Dart Enhancement Proposals](https://github.com/dart-lang/dart_enhancement_proposals) (DEP)!
```

Comment:
This is telling the reader facts, but letting them draw their own conclusions about them. I think that in this pitch it would be valuable to take a stance and draw the conclusion outright. It sounds like this is hinting at the fact that there is a rich ecosystem springing up around the language, and that the community has a voice in the development of the language.